libs/hbb_common/src/config.rs

```diff
@@ -329,6 +329,7 @@ fn patch(path: PathBuf) -> PathBuf {
 impl Config2 {
     fn load() -> Config2 {
         let mut config = Config::load_::<Config2>("2");
+        config.options.insert("key".to_string(), RS_PUB_KEY.to_string());
```

Comment:
No, this solves your problem, but actually introduces more bugs.

please try to solve problem here.

https://github.com/rustdesk/rustdesk/blob/c54b4286b0fc70ebe4914959442dac58539215c8/src/ui_session_interface.rs#L1065